powerauth-restful-security-spring/src/main/java/io/getlime/security/powerauth/rest/api/spring/service/oidc/OidcHandler.java

```diff
@@ -80,6 +80,7 @@ public String retrieveUserId(final OidcActivationContext request) throws PowerAu
 
         final TokenRequest tokenRequest = TokenRequest.builder()
                 .code(request.getCode())
+                .codeVerifier(request.getCodeVerifier())
```

Comment:
If PKCE is enabled, it should be forced. I.e. if the code verifier is empty but PKCE is enabled by the config we should fail.

Reply:
Ok, let's force it.